Added a new medication, Miner's Salve

lordpidey opened the pull request:
Adds a new medication, which will give doctors a new long term healing option, miners a way to make a medication on mining station, and will have some nefarious uses too for traitors.

lordpidey committed:
Added a new medication, Miner's Salve

ojacobson commented on code/modules/reagents/Chemistry-Recipes/Medicine.dm:
```diff
@@ -57,6 +57,22 @@
+	id = "mine_salve"
+	result = "mine_salve"
+	required_reagents = list("oil" = 1, "water" = 1, "iron" = 1)
+	mix_message = "The fine black particles dissolve into the oily substance."
```
Thematic niggle: these two messages (his, and the one in `mine_salve2`, below, are considerably more detailed than the other reaction messages. Keep in mind that these can be seen from several tiles away in some circumstances: does it make sense for someone at the chem door to be observing fine particles in anything?

I don't necessarily have an improved suggestion for you, but it stood out for me.

Jordie0608 commented on code/modules/reagents/Chemistry-Reagents/Medicine-Reagents.dm:
```diff
@@ -223,6 +223,30 @@
+	metabolization_rate = 0.4 * REAGENTS_METABOLISM
+
+/datum/reagent/medicine/mine_salve/on_mob_life(mob/living/M)
+	if(istype(M, /mob/living/carbon))
```
There's an IsCarbon() check which does the same as this and is better to use for some reason.

ojacobson commented on code/modules/mob/living/carbon/human/species.dm:
```diff
@@ -737,6 +738,8 @@
 					icon_num = 4
 				if(damage > (comparison*4))
 					icon_num = 5
+				if(H.hal_screwyhud == 5)
```
I like this. We have another med that gives misleading "you are healthy" indicators (Aranesp), and it's one of the more entertaining things to dose someone with.

ojacobson commented:
I'm not sure how I feel about a recipe that suggests that oil (an easy-to-make early tier reagent) and plasma (notionally complicated, but in fact available from the chem vendor) are interchangeable. Given this recipe, I'll be curious if anyone keeps making saline-glucose mix.

Chocobro commented:
how to do your first PR right

seriously this guy did his own changelog yml and actually bothered to ask in coderbus when he had trouble.

looks good to me. :+1:

ojacobson commented:
> seriously this guy did his own changelog yml and actually bothered to ask in coderbus when he had trouble.

also this. :+1: :heavy_multiplication_x:  :100:

WJohn commented:
Do we really need another tricord?

lordpidey committed:
Removed mixing messages as they didn't make sense.  Also changed istype checks to iscarbon checks.

lordpidey commented:
WJohn, it's a very slow and long lasting medication with unique side effects.  The side effects could easily be used by a traitor.

ojacobson, being able to use plasma for the recipe allows it to be made entirely on the mining station, hence it's name.  Throw a sheet of metal, plasma, and 20 units of water in a beaker and you've got some medication.

phil235 commented:
It's not exactly slow, its meabolization rate is between salglu and omnizine. And its healing power is lower than omnizine.

Iamgoofball commented:
This isn't a tricord, WJ.

This is good. I like it. It has an upside and a downside. It can be used for traitoring or for healing.

:+1:

SconesC commented:
Miners don't have a grinder though

milocv commented:
Looks nice. I think there should be a lot more "ghetto chemistry" options.

Jalleo commented:
This is probably one of the best medicines to hear about.

What are the side effects?

Thunder12345 commented:
It makes you think you're instantly at full health. Consider the effect of adding a little of that to a poison mix. They'll never even know they're dying...

AlexanderUlanH commented:
'Salve' doesn't work as a touch application.  Might want to rename it.
Also, like Scones said, you can't just dissolve a sheet of metal or plasma in a cup.  If you want miners to have access to this, you'll have to figure out some other way for them to get reagents.

lordpidey commented:
Iamgoofball, I agree, there should be mix messages for everything, but wouldn't that be better to do in another PR?

And I could have sworn there was a grinder on the mining station.  I guess I should have checked first.

ojacobson commented:
@lordpidey solution: add grinder to the kitchen in the mining station! :D

lordpidey committed:
Gave Miner's salve a slight side effect when ingested, to show that it's designed to be applied, not ingested.
Changed secondary recipe of Miner's Salve to be a sheet of iron+ a sheet of plasma + a twinkie in a grinder.  That seems more "ghetto chemistry"
Added grinder to mining station.

lordpidey commented:
I've listened to the suggestions on here.  
I've added a minor ingestion side effect to indicate that it's meant to be applied, just a short stun and weakness, no damage.

I've added a grinder to the mining station's kitchen.

I've changed the secondary recipe to be more "ghetto" chemistry.  It is now a sheet metal + a plasma sheet + a twinkie in a grinder.

SconesC commented:
this is enabling a really horrific sleeper op miner strat that has nothing to do with the reagent in questions and i love it

lordpidey commented:
What strategy would that be?

As far as I know the only chemicals that can be gotten in the mining outpost with a grinder are
Gold, silver, iron, plasma, sillicon, uranium, water, sugar, nutrament, space beer, and nicotine.

The only existing recipe that jumps out at me is EMP.
Without stabilizing agent, EMP runs will be quite difficult.

Rinoka commented:
Traitors can get poison boxes and sleepy pens.

This lets you delude people into thinking nothing's wrong and they'll be fine until they go unconscious and die because you pricked them and ran. And I love it.
